src/config.py

```diff
@@ -13,6 +13,7 @@ class Config(object):
     USE_GOOGLE_AUTH = os.getenv('AUTH_HEADER_NAME') is None
     SKIP_AUTH = os.getenv('SKIP_AUTH').lower() in ['1', 'true', 't']
     BEHIND_PROXY = os.getenv('BEHIND_PROXY', '0').lower() in ['1', 'true', 't']
+    HTTPS_REDIRECT_URL = os.getenv('HTTPS_REDIRECT_URL', 'https://localhost:8443')
```

Comment:
Currently, the respective env vars in staging and production are:
`HTTPS_REDIRECT_URL=https://gogo-stg.dev1-eks.nextdoor-test.com`
`HTTPS_REDIRECT_URL=https://go.corp.nextdoor.com`